src/test/java/integration/ImportDocumentApiTest.java

```diff
@@ -1,9 +1,20 @@
 package integration;
 
 import com.laserfiche.repository.api.clients.EntriesClient;
-import com.laserfiche.repository.api.clients.impl.model.DeleteEntryWithAuditReason;
+import com.laserfiche.repository.api.clients.impl.model.*;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.net.ServerSocket;
```

Comment:
Nitpick: it seems we don't need this import.

Reply:
Fixed.